report/completeness.html

```diff
@@ -0,0 +1,147 @@
+        if (mm < 10)
+            mm = "0" + mm;
+
+        let cur_day = aaaa + "-" + mm + "-" + gg;
+
+        let hours = date.getHours()
+        let minutes = date.getMinutes()
+        let seconds = date.getSeconds();
+
+        if (hours < 10)
+            hours = "0" + hours;
+
+        if (minutes < 10)
+            minutes = "0" + minutes;
+
+        if (seconds < 10)
+            seconds = "0" + seconds;
+
+        return cur_day + " " + hours + ":" + minutes + ":" + seconds;
+    }
+
+    async function getStatus(_country, _countryValue) {
+        let response = await fetch("completeness_" + _countryValue + ".json").then((response) => {
+            if (response.ok) {
+                return response.json();
+            }
+            throw new Error('Something went wrong');
+        })
+            .then((responseJson) => {
+                document.getElementById("countryHeader").innerHTML = `Completeness report for ${_country}`;
+                let completenessTable = `
+            <h2>Available datasets</h3>
+            <table id="compTable" class="sortable">
+                    <tr class="header">
+                        <th>Dataset</th>
+                        <th>Source</th>
+                        <th>Category</th>
+                        <th>Theme</th>
+                        <th>Type</th>
+                        <th>Acquired Datasets - Date</th>
+                    </tr>`
+
+                for (let i = 0; i < responseJson.length; i++) {
+                    completenessTable +=
+                        `<tr class=${responseJson[i].class}>
+                        <td>${responseJson[i].dataset}</td>
+                        <td>${responseJson[i].source}</td>
+                        <td>${responseJson[i].category}</td>
+                        <td>${responseJson[i].theme}</td>
+                        <td>${responseJson[i].type}</td><td>`
+                    responseJson[i].acquiredDatasets.forEach((fileName, idx) => {
+                        completenessTable +=
+                            `<table><tr>
+                            <td width="50%" class="tdAlign">${fileName}</td>
+                            <td width="50%" class="tdAlign">${formatDate(responseJson[i].dateTime[idx])}</td>
+                            </tr></table>
+                            `
+                    });
+                    completenessTable += "</td></tr>"
+                }
+
+                completenessTable += "</table>";
+
+                const completeness = document.getElementById("completenessTable");
+                completeness.innerHTML = completenessTable;
+            })
+            .catch((error) => {
+                let errorTxt = "No data for selected country!"
+                document.getElementById("countryHeader").innerHTML = selectCountry;
+                document.getElementById("completenessTable").innerHTML = `<h1 id="countryHeader">${errorTxt}</h2>`;
+                console.log(errorTxt + error)
```

Comment:
so you are about to give info about no data for selected country for every error you meet. Without any dependency on error type. it may cause some strange behaviour when firstly you say on FE that there is no data and then the other error text. 
 
If I understand correctly, even if just one field is absent, you will answer in that way.

Reply:
made changes for
if can't reach file will return no data for country
if any other error occurs something went wrong. and logs on console